src/lib/components/css/reset.css

```diff
@@ -1,3 +1,7 @@
+html {
+	scroll-behavior: smooth;
+}
```

Comment:
Okay, maybe then it could be ussed just for defined context or params for routes? For now i just moved this to `src/routes/+page.svelte`. It's a little bit weird though that this quirky scrolling issue arises after globally applying this rule...

For now i just moved this to Home page component — 86626e177de562e7f798562ff2a602f7977b21f1